src/main/java/util/ApplicationUtils.java

```diff
@@ -0,0 +1,22 @@
+package util;
+
+import org.commcare.api.persistence.SqlSandboxUtils;
+
+/**
+ * Utility methods for dealing with Applications
+ */
+public class ApplicationUtils {
```

Comment:
Wasn't sure if this was the best place for this, but couldn't find anything else